_components/range-slider/range-slider.md

```diff
@@ -14,6 +14,14 @@ type: component
 maturity: alpha
 id_custom: range
 lead: A range slider allows users to choose an approximate number from a range.
+implementation:
+  initProps:
+    - property: "`data-text-unit`"
+      description: "Adds a unit to screen reader callouts. The unit will be read after the current value."
```

Comment:
Thinking it might be beneficial to add an example. Here's one possibility:
```suggestion
      description: 'Adds a unit to screen reader callouts. The unit will be read after the current value. For example, adding a value of "stars" enables a readout like "3.5 stars of 5".'
```

Reply:
Resolved in [16a0ea7](https://github.com/uswds/uswds-site/pull/2310/commits/16a0ea7739e49bfa25480dc73bca6931890624d9)